SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/WorkerCoordinator/SpatialConfig/cloud_launch_sim_player_deployment.json

```diff
@@ -0,0 +1,40 @@
+{
+  "template": "sim_players",
+  "world": {
+    "chunkEdgeLengthMeters": 50,
+    "snapshots": {
+      "snapshotWritePeriodSeconds": 0
+    },
+    "dimensions": {
+      "xMeters": 300,
+      "zMeters": 300
+    }
+  },
+  "load_balancing": {
+    "layer_configurations": [
+      {
+        "layer": "SimulatedPlayerCoordinator",
+        "rectangle_grid": {
+          "cols": 10,
+          "rows": 1
+        }
+      }
+    ]
+  },
+  "workers": [
+    {
+      "worker_type": "SimulatedPlayerCoordinator",
+      "flags": [
+		    {
+          "name": "coordinator_start_delay_millis",
+          "value": "10000"
```

Comment:
Any documentation on why we need a start delay?

Reply:
This is to prevent the simulated clients from connecting too soon after the target deployment is up. It adds some buffer to allow the target deployment to properly start and initialise